openeo/util.py

```diff
@@ -657,7 +670,15 @@ def normalize_crs(crs: Any, *, use_pyproj: bool = True) -> Union[None, int, str]
         Other data structures will not be accepted.
 
     :param crs: data structure that encodes a CRS, typically just an int or string value.
-        If the ``pyproj`` library is available, everything supported by it is allowed
+        If the ``pyproj`` library is available, everything supported by it is allowed.
+
+        Integers always refer to the EPSG code that has that number assigned.
```

Comment:
I would move these additional paragraphs in the `param crs` section to the main body of the function docs, as it describes/illustrates the core behavior. Now there seems quite some duplication

Reply:
Done, updated accordingly.